template.yaml

```diff
@@ -745,6 +745,32 @@ Resources:
               "Type": "Task",
               "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-Set-Account",
               "InputPath": "$",
+              "Next": "AddAccountToAlertDestinations"
+            },
+            "AddAccountToAlertDestinations": {
+              "Type": "Task",
+              "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-Add-Account-To-Destination",
```

Comment:
Do we need alerts for this account?

Reply:
This change does not require alerts. I will remove this